tools_for_todoist/services/calendar_to_todoist.py

```diff
@@ -90,19 +92,39 @@ def _process_cancelled_event(self, calendar_event):
             todoist_item = self.todoist.get_item_by_id(todoist_id)
             if todoist_item is not None:
                 self.todoist.delete_item(todoist_item)
+    
+    def _process_updated_event(self, calendar_event):
```

Comment:
Right now on first sync when we sync calendar events we ignore those that already have a linked todoist item. Since there could have been changes while the app was down, we should also updated them there if their due date should be fixed.